code/modules/cargo/bounties/science.dm

```diff
@@ -90,56 +115,45 @@
 	name = "Drone Control Circuitboard"
 	wanted_types = list(/obj/item/circuitboard/drone_control)
 
-/datum/bounty/item/science/circuitboard/cloning
-	name = "Cloning Control Circuitboard"
-	wanted_types = list(/obj/item/circuitboard/cloning)
-
 /datum/bounty/item/science/circuitboard/powermonitor
 	name = "Power Monitor Circuitboard"
 	wanted_types = list(/obj/item/circuitboard/powermonitor)
 
 /datum/bounty/item/science/battery
 	name = "Heavy-Duty Power Cell"
-	description = "%COMPNAME has requested some power cells to fill their supply closet. Please charge them first."
-	reward = 3000
-	required_count = 5
+	description = "%COMPNAME has requested some power cells to fill their supply closet. Please fully charge them first."
+	reward_low = 35
+	reward_high = 45
+	required_count = 4
+	random_count = 1
 	wanted_types = list(/obj/item/cell/apc)
 
+/datum/bounty/item/science/battery/applies_to(obj/O)
```

Comment:
Instead of having:

```
/datum/bounty/item/science/battery/applies_to(obj/O)
var/obj/some/type/T = O
if!(istype(T, /obj/some/type))
    return FALSE
T.logic()
more_logic(T)
```

Everywhere, we could do:

```
/datum/bounty/item/science/battery/applies_to(var/obj/some/type/O)
if(!istype(O))
    return FALSE
O.whatever()
some_logic(O)
```

So stuff is more simply typecast. 
Ofc it's equivalent and doesn't matter too much.